### PR TITLE
chore: avoid recursion in `format`/`lint`

### PR DESCRIPTION
**Pull request by brettz9**
**What is the purpose of this pull request?**

- [x] Other, please explain:

**What changes did you make? (Give an overview)**

Fixes an npm linting script

### CODE REVIEW

**Comment by voxpelli** [comment]
What is the actual fix here? :)

There is one difference here, it's that the report flag is not in the format script?

**Comment by brettz9** [comment]
Sorry, this is now being called recursively. `format` was calling `lint`. `lint` calls `lint:*` which includes `lint:eslint-docs` which runs `format` again. Calling just `eslint` avoids the circle.

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "Aadit M Shah <aaditmshah@aadit.codes> (https://aadit.codes/)"
   ],
   "scripts": {
-    "format": "prettier --write . && npm run lint -- --fix",
+    "format": "prettier --write . && eslint . --fix",
```
Comment (voxpelli) [inline package.json:25]: How about?

```suggestion
    "format": "prettier --write . && npm run lint:js -- --fix",
```

**Review by voxpelli — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (voxpelli) [reply package.json:25]: Never mind, I noticed that it's breaking main branch, so exact fix isn't very important :)

**Pull request merged**